assets/dev/scss/common/common.scss

```diff
@@ -5,7 +5,7 @@
 @import "dialog-common";
 @import "dialog-templates";
 @import "notifications";
-@import "app/modules/site-editor/assets/scss/admin/admin-bar";
+@import "../../../../app/modules/site-editor/assets/scss/admin/admin-bar";
```

Comment:
We get an error if we import without the full path.

Should we copy these files to the Pro repo? Or is there another way how we can handle this?